pkg/provider/webhook/webhook.go

```diff
@@ -26,7 +26,7 @@ import (
 	"strings"
 	tpl "text/template"
 
-	"github.com/Masterminds/sprig"
+	"github.com/Masterminds/sprig/v3"
```

Comment:
this might seem weird, but `v3` is api-compatible with `v2`. This is just to consolidate imports.